util/file.go

```diff
@@ -140,3 +142,44 @@ func mkUniq(prefix string, create func(string) error) (string, error) {
 
 	return "", err
 }
+
+// GetVersion returns the version of the binary file. It distinguishes between two cases:
+// a) if the current commit is the same commit of the last tag it returns the tag's name.
+// b) if the current commit differs from the commit of the last tag it returns
+// the short commit hash of the actual commit appended to the "dev" and date of commit.
+func GetVersion() (string, error) {
+	var err error
+	versionCmd := exec.Command("git", "describe", "--tag", "--abbrev=0")
```

Comment:
This makes the binary depending on being able to call `git`, which is not desirable. The version number should be determined at build time and then "burned" into the binary.